docker/Makefile

```diff
@@ -0,0 +1,45 @@
+NAMESPACE ?= puppet
+git_describe = $(shell git describe)
+vcs_ref := $(shell git rev-parse HEAD)
+build_date := $(shell date -u +%FT%T)
+hadolint_available := $(shell hadolint --help > /dev/null 2>&1; echo $$?)
+hadolint_command := hadolint --ignore DL3008 --ignore DL3018 --ignore DL4000 --ignore DL4001
+hadolint_container := hadolint/hadolint:latest
+pwd := $(shell pwd)
+export BUNDLE_PATH = $(pwd)/.bundle/gems
+export BUNDLE_BIN = $(pwd)/.bundle/bin
+export GEMFILE = $(pwd)/Gemfile
+
+version = $(shell echo $(git_describe) | sed 's/-.*//')
```

Comment:
Will this overwrite the 1.23.0 (for example) tag every time we push a commit until we've tagged 1.24.0?

Reply:
With the existing distelli config I have in this PR, yes, but with the caveat that it will still contain bolt 1.23.0 since this container is installing from packages.

The makefile could also be executed as part of the bolt publishing job if you only wanted to push the container at release time.